Jenkinsfile

```diff
@@ -58,7 +58,11 @@ pipeline {
                     if (body.contains("[test cassandra]") || title.contains("cassandra:")) {
                         env.testProvider = "cassandra"
                     } else if (body.contains("[test ceph]") || title.contains("ceph:")) {
-                        env.testProvider = "ceph"
+                        if (body.contains("[test full]")) {
+                          env.testProvider = "ceph"
+                        } else {
+                          env.shouldBuild = "false"
```

Comment:
Also how about adding a comment about why we disable the build

Reply:
Added.

I see it is skipped after removing `[test full]` https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/PR-6664/6/pipeline